### DISCUSSION TITLE
Plans for linux-arm64 support?

### DISCUSSION THREAD

**Discussion by xemle** [getgauge/gauge#2035] [Ideas]
Hi.

I am wondering why linux-arm64 is missing? Are there any obstacles not to do it or is there no demand? I would like to use gauge-js on my CI pipeline running on a Raspberry PI 4. Currently I would not require taiko...

And thank you for providing such a great test tool. Love the simplicity via markdown! Great job!

**Comment by sriv** [comment]
Hi, thank your for the compliments!

Couple of reasons for not having linux_arm64 support yet - main one is that no one has asked for it until now, but also that we do not have arm64 machines to run and test these on.

That said, you can always to a `go get` or install from source on an arm64 machine, golang will build the binary for you.

If you still prefer a prebuilt binary, please consider raising a pull request. I'll be happy to guide you (if you need any help) and merge/release it.

**Comment by xemle** [comment]
Thank you for your quick reply. If possible I would prefer a official linux-arm64 version ;-)

As far as I understood, only platformEnvs in [make.go](https://github.com/getgauge/gauge/blob/master/build/make.go#L158) needs an additional entry and through the github release workflow the binary will be compiled and the npm package will download the correct version out of the box, too.

Did I miss something? If not, I will send an PR...

**Reply by sriv** [reply]
I think that should be enough, given that arm64 architecture was added for macOS recently - https://github.com/getgauge/gauge/pull/1998

**Comment by xemle** [comment]
Same would be for screenshot, right?

**Reply by sriv** [reply]
Yes, the build scripts are similar

**Comment by xemle** [comment]
See my PR #2036 for linux-arm64 support

**Reply by sriv** [reply]
Thanks, will look into this.
